changelog.md

```diff
@@ -1,5 +1,151 @@
+
+- Add: Buttons to change picker preview image height. (#14373)
+- Add: Show and sort online content version in content list. (#14355)
+- Add: Hover on graph legend to highlight line.
+- Add: Allow separate expansion of town buildings and roads in scenario editor. (#14341)
+- Add: [Script] Cloning ScriptList
+- Add: [Script] Framework for cloning selected ScriptObject
+- Add: Badge dropdown filters and configuration.for picker and vehicle windows.
+- Add: Configuration of NewGRF badges for picker and vehicle windows.
+- Add: Generic "configure list/view" vertical ellipsis icon.
+- Add: [Script] GSBaseStation::GetOwner (#13406)
+- Add: [Script] ScriptVehicleList_Waypoint (#13456)
+- Add: [NewGRF] All callbacks returning D0xx strings, have now the option to return any string id via register 0x100.
+- Add: [NewGRF] Special value 0x7FFE for VarAction2 results specifying 'return calculated result'.
+- Add: [NewGRF] Special value 0x7FFF for Action2 references specifying explicit 'callback/sprite-resolving failed'.
+- Add: [NewGRF] Add purchase list Action3 CID for houses, industries, industry tiles, airports and airport tiles. (#14121)
+- Add: "Many random towns" button in scenario editor now prompts for the number of towns, with defaults based on new game settings (#14158)
+- Add: [BaseSet] Allow basesets to set minor and patch versions in obg/obs/obm files.
+- Add: Limit height of settings description, and add scrollbar. (#14102)
+- Add: [NewGRF] Provide random bits in var10 of house callback 1C 'construction stage changed'. (#14095)
+- Add: [NewGRF] Station/roadstop animation-triggers 'tile loop' (bit 7) and 'path reservation' (bit 8). (#14080)
+- Change: Improve sprite ellipsis appearance. (#14404)
+- Change: New company face definition system and UI. (#14319)
+- Change: Simplify sprite cache memory management
+- Change: Tweak layout of music player widgets.
+- Change: Consistent toolbar sizes with dedicated NWidgetParts. (#14344)
+- Change: Reflow Textfile window content incrementally.
+- Change: Don't measure group name widths in company livery window. (#14291)
+- Change: [Script] Don't allow scripts to instantiate ScriptEvent (#14276)
+- Change: [Script] ScriptVehicleList_Station accepts an optional VehicleType parameter (#14260)
+- Change: Remove Apply button from NewGRF config window when unneeded. (#14254)
+- Change: Remove extra close buttons from some windows. (#14124)
+- Change: Move OpenTTD version to intro viewport. (#14233)
+- Change: [NewGRF] Increase the textstack for all callbacks to 16 registers.
+- Change: Rearrange buttons in the intro screen, remove unnecessary ones.
+- Change: Remove the limit of 20 parameters to a Script Text string. (#14193)
+- Change: Use same padding as NewGRF parameters for AI/GS parameters lists. (#14118)
+- Change: ParseInteger now checks for trailing junk chars.
+- Change: [NewGRF] If Action123 does not resolve in a valid SpriteSet, prefer drawing the default sprite instead of an invalid sprite.
```

Comment:
Might be `Fix:` ?

```suggestion
- Change: [NewGRF] If Action123 does not resolve in a valid SpriteSet, prefer drawing the default sprite instead of an invalid sprite. (#14143)
```